src/fabric/src/fabric2_fdb.erl

```diff
@@ -647,8 +649,12 @@ write_doc(#{} = Db0, Doc, NewWinner0, OldWinner, ToUpdate, ToRemove) ->
         atts = Atts
     } = Doc,
 
-    % Doc body
+    % Fetch the old doc body for the mango hooks
+    OldWinnerDoc = if OldWinner == not_found -> not_found; true ->
```

Comment:
The `fabric2_index:db_updated` hook only sends the Db info. I can switch it to use the `fabric2_db_plugin:after_doc_write`?

Reply:
Just the DbName it looks like. Also looking back I just realized its outside of the transaction as well so isn't nearly as close as I was thinking. We don't want to move that to after_doc_write since it'd cause extra contention around its ets tables.

Lets stick with the current approach. I was thinking they were a lot closer together in the write path is all.